pyomo/contrib/multistart/test_multi.py

```diff
@@ -31,7 +31,11 @@ def test_as_good_with_iteration_rand(self):
         for i in range(10):
             m2 = build_model()
             SolverFactory('multistart').solve(m2, iterations=10)
-            self.assertTrue((value(m2.obj.expr)) >= (value(m.obj.expr) - .001))
+            m_objectives = m.component_data_objects(Objective, active=True)
+            m_obj = next(m_objectives, None)
+            m2_objectives = m2.component_data_objects(Objective, active=True)
+            m2_obj = next(m2_objectives,None)
+            self.assertTrue((value(m2_obj.expr)) >= (value(m_obj.expr) - .001))
```

Comment:
I'm not sure that function would apply in this situation since the test is trying to check that it does no worse than the standart solver. The multistart solver could give a result much larger than the standard solver.

Reply:
Ok, that makes sense, I misunderstood the intent of the test.

Reply:
A comment to this effect would be useful, as I misread the intent of the test in the same way @blnicho did.